### ISSUE TITLE
Training dataset used in Reported mAP under README.md

### ISSUE THREAD

**Issue by JWLee89** [ultralytics/yolov5#7896]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello, 

In the `README.md`, it states

> All checkpoints are trained to 300 epochs with default settings. Nano and Small models use [hyp.scratch-low.yaml](https://github.com/ultralytics/yolov5/blob/master/data/hyps/hyp.scratch-low.yaml) hyps, all others use [hyp.scratch-high.yaml](https://github.com/ultralytics/yolov5/blob/master/data/hyps/hyp.scratch-high.yaml).

I noticed that the default dataset is `coco128`. Does this mean that the checkpoints are trained to 300 epochs using `coco128` (not using coco2017 training set, which is much larger)? 

I just wanted to double check so that in the future, I report accurate figures using the appropriate dataset.  

Would explicitly mentioning the training dataset used to generate the reported figures in `README.md` help improve clarity? 

Are these figures reported based off a single run or averaged over multiple runs (with varying random seed values for reproducibility)?

Thank you for taking the time to read this!

**Comment by glenn-jocher** [comment]
@JWLee89 official models are tracked here including commands to reproduce: https://wandb.ai/glenn-jocher/YOLOv5_v61_official

**Comment by JWLee89** [comment]
@glenn-jocher Thank you for the link. 

Just bookmarked the link and made notes on what information is available.

**Comment by glenn-jocher** [comment]
@JWLee89 you're welcome! If you have any more questions or need further assistance, feel free to ask. Good luck with your YOLOv5 endeavors!
